k8s.io/test.py

```diff
@@ -66,6 +66,11 @@ def test_go(self):
 
     # TODO: external go-get ???
 
+    def test_apt_test(self):
+        # FIXME: https://apt.kubernetes.io is not on the cert
+        base = 'http://apt.kubernetes.io'
+        self.assert_redirect(base, 'http://storage.googleapis.com/k8s-releases/apt/')
```

Comment:
will do

On Tue, Aug 23, 2016 at 11:59 AM, Ryan Hitchman <notifications@github.com>
wrote:

> In k8s.io/test.py
> <https://github.com/kubernetes/k8s.io/pull/4#discussion_r75929730>:
>
> > @@ -66,6 +66,11 @@ def test_go(self):
> >
> >      # TODO: external go-get ???
> >
> > +    def test_apt_test(self):
> > +        # FIXME: https://apt.kubernetes.io is not on the cert
> > +        base = 'http://apt.kubernetes.io'
> > +        self.assert_redirect(base, 'http://storage.googleapis.com/k8s-releases/apt/')
>
> maybe do self.assert_redirect(base + '/$id', 'http://storage.googleapis.
> com/k8s-releases/apt/$id', id=rand_num()) to test that paths are
> redirecting properly.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/k8s.io/pull/4/files/7c8000e9c6131f670e3457732d46204fdaae20f4#r75929730>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVCiRNdXNYVScHSIsODwv7SIAqScuks5qi0L-gaJpZM4JqiuI>
> .
>